src/common/services/api/order.service.spec.js

```diff
@@ -711,7 +711,23 @@ describe('order service', () => {
 
       self.$httpBackend.expectPOST(
         'https://give-stage2.cru.org/cortex/enhancedpurchases/orders/crugive/me3gkzrrmm4dillegq4tiljugmztillbmq4weljqga3wezrwmq3tozjwmu=?followLocation=true',
-        { 'cover-cc-fees': false }
+        { 'cover-cc-fees': false, 'radio-call-letters': null }
+      ).respond(200, purchaseResponse)
+
+      self.orderService.submit()
+        .subscribe((data) => {
+          expect(data).toEqual(purchaseResponse)
+        })
+
+      self.$httpBackend.flush()
+    })
+
+    it('should send the radio cover letters to the server', () => {
```

Comment:
One thing I've noticed with these tests that follow the pattern:
```
someService.function()
  .subscribe((data) => {
    expect(stuff)
})
```
Is that we need to use the [done](https://jestjs.io/docs/asynchronous) callback in order for it to actually be testing something.

Reply:
Would you want me to go through the other tests in this file and add `done()` to them?

Reply:
You can if you want.